sql/core/src/main/scala/org/apache/spark/sql/DataFrameNaFunctions.scala

```diff
@@ -394,10 +394,11 @@ final class DataFrameNaFunctions private[sql](df: DataFrame) {
   }
 
   private def fillMap(values: Seq[(String, Any)]): DataFrame = {
+    var resolved: Map[String, Any] = Map()
```

Comment:
I would you a mutable map assigned to a `val`.

Reply:
@HyukjinKwon done.